pkg/kudoctl/packages/verifier/template/verify_render.go

```diff
@@ -38,10 +38,18 @@ func templateCompilable(pf *packages.Files) verifier.Result {
 
 	engine := renderer.New()
 	for k, v := range pf.Templates {
-		_, err := engine.Render(k, v, configs)
+		// Render the template
+		s, err := engine.Render(k, v, configs)
 		if err != nil {
 			res.AddErrors(err.Error())
 		}
+
+		// Try to parse rendered template as valid YAML
```

Comment:
Nit: This isn't quite right. This checks that the renderer template omits valid YAML of Kubernetes objects.
```suggestion
		// Try to parse rendered template as valid Kubernetes objects
```